1458TeamCode/src/main/java/frc/robot/subsystems/Module.java

```diff
@@ -0,0 +1,95 @@
+package frc.robot.subsystems;
+
+import com.ctre.phoenix6.controls.DutyCycleOut;
+import com.ctre.phoenix6.controls.PositionVoltage;
+import com.ctre.phoenix6.controls.VelocityVoltage;
+import com.ctre.phoenix6.hardware.CANcoder;
+import com.ctre.phoenix6.hardware.TalonFX;
+
+import edu.wpi.first.math.controller.SimpleMotorFeedforward;
+import edu.wpi.first.math.geometry.Rotation2d;
+import edu.wpi.first.math.kinematics.SwerveModulePosition;
+import edu.wpi.first.math.kinematics.SwerveModuleState;
+import frc.robot.lib.math.Conversions;
+import frc.robot.lib.util.SwerveModuleConstants;
+import frc.robot.Constants;
+import frc.robot.CTREConfigs;
+import frc.robot.Robot;
+
+public class Module {
+    public int moduleNumber;
```

Comment:
change it to enum with FrontLeft, FrontRight and so on

Reply:
Not sure exactly what needs to be changed, the latest push doesn't change anything relating to this